pkgs/by-name/ex/exercism/package.nix

```
@@ -22,8 +24,17 @@ buildGoModule rec {
 
   subPackages = [ "./exercism" ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
   passthru.updateScript = nix-update-script { };
 
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

```suggestion
  postInstall = ''
```
The completions are already generated as files, they can always be installed.

---

@FliegendeWurst indeed, updated!